src/test/fuzz/txorphan.cpp

```diff
@@ -38,6 +38,7 @@ FUZZ_TARGET(txorphan, .init = initialize_orphanage)
 
     TxOrphanage orphanage;
     std::vector<COutPoint> outpoints; // Duplicates are tolerated
+    outpoints.reserve(200'000);
```

Comment:
Are you saying this will affect fuzzing behavior? Does it look under the hood to check how many items to create? I wasn't aware of that and find it odd.
What value do you recommend here instead of the worst case?